### PR TITLE
Windows support

### LINKED ISSUE #6: Windows support

### LINKED ISSUE #7: Create package with bundled libmpv

### PR DESCRIPTION
**Pull request by fsobolev**
image

For some reason MPV can't find yt-dlp in PATH, so I had to place the binary in config dir (`%APPDATA%/mpv`) for the examples to work, and also yt-dlp seems to be very slow on Windows (it takes *a lot* of time to load a video from youtube, but in WSL it's fine). Other than that the examples work as expected.

Closes #6 
Closes #7 

TODO:
- [x] Update docs
- [x] Create `MPVSharp.Bundled` package with libmpv built-in

### CODE REVIEW

**Comment by fsobolev** [comment]
Okay, seems like I made working package with bundled libmpv. Dll file is from [this repo](https://github.com/shinchiro/mpv-winbuild-cmake), I used latest release. I didn't find anything similar pre-built for Linux, and building this myself seems to be very time-consuming, and considering that installing and adding libmpv to linux packages is much easier than doing the same on windows, I think it's fine to have Bundled package windows-only, it only really preferred there IMO. The library here seems to be complete, with GPL parts, so I set license for Bundled package to be GPL too (I'm not sure that copyleft really works here, but I think it's better to assume it does).

For testing I modified OpenGL example's csproj to have this:

```
  <PropertyGroup> 
<RestoreSources>$(RestoreSources);../../Nickvision.MPVSharp/bin/Debug;../../Nickvision.MPVSharp.Bundled/bin/Debug;https://api.nuget.org/v3/index.json</RestoreSources>
  </PropertyGroup>
  
  <ItemGroup>
    <PackageReference Include="OpenTK" Version="4.7.7" />
    <PackageReference Include="Nickvision.MPVSharp.Bundled" Version="2023.7.2" />
  </ItemGroup>
```

And it worked.

I first tried to have ProjectReference instead of PackageReference in Bundled, but it was causing NU1102 error when build the example for some reason.

File: Nickvision.MPVSharp.Bundled/Nickvision.MPVSharp.Bundled.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <PackageLicenseExpression>GPL-2.0-or-later</PackageLicenseExpression>
+    <Copyright>(c) Nickvision 2021-2023</Copyright>
+    <PackageProjectUrl>https://nickvision.org</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/NickvisionApps/MPVSharp</RepositoryUrl>
+    <PackageReleaseNotes>- Initial Release (uses libmpv build 20230901)</PackageReleaseNotes>
+    <PackageIcon>logo-r.png</PackageIcon>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Nickvision.MPVSharp" Version="2023.7.2" />
```
Comment (nlogozzo) [inline Nickvision.MPVSharp.Bundled/Nickvision.MPVSharp.Bundled.csproj:23]:
```xml
<ProjectReference Include="..\Nickvision.MPVSharp\Nickvision.MPVSharp.csproj" />
```

**Review by nlogozzo — CHANGES_REQUESTED** [review]
Let's use `ProjectReference` here that way when we build the nuget for `Bundled` we don't need to wait for the regular one to upload to nuget then update the Package ver then build the Bundled nuget...

Everything else looks great tho!

File: Nickvision.MPVSharp.Bundled/Nickvision.MPVSharp.Bundled.csproj (hunk 1 above)
Comment (fsobolev) [reply Nickvision.MPVSharp.Bundled/Nickvision.MPVSharp.Bundled.csproj:23]: okay, let me try again, I will try to find why I get NU1102 error when using ProjectReference

Comment (fsobolev) [reply Nickvision.MPVSharp.Bundled/Nickvision.MPVSharp.Bundled.csproj:23]: Found why I was getting the error. It's because I was removing `../../Nickvision.MPVSharp/bin/Debug` in RestoreSources in example's csproj. When dotnet packs Bundled it doesn't include MPVSharp, but creates a dependency with matching version, so the difference between PackageReference and ProjectReference is that for ProjectReference we just don't need to specify a version.

https://stackoverflow.com/a/65803800
> By default, when you use ProjectReference to Reference Project A into Project B under new-sdk project, it actually views the Project A as a nuget dependency.

But this seems to not be a problem, looks like it will be possible to upload Bundled even without uploading MPVSharp (but the users will not be able to install it).
https://stackoverflow.com/a/70764466
> dotnet pack and nuget pack assume that you're going to also be packing each of those projects as packages as well, and writes the project reference as a package dependency at the same version of the dependent project package. So, if you pack projB @ v1.2.3 the package will have a dependency reference to projA @ v1.2.3. If you don't pack projA @ v1.2.3 or you don't publish that package (because maybe there weren't any changes to it), consumers of projB @ v1.2.3 will fail the install because nuget won't find projA @ v1.2.3.

**Review by nlogozzo — APPROVED** [review]
